desktop/assets/texts/tr-TR/settings.ftl

```diff
@@ -51,6 +52,7 @@ scale-mode-noscale = Ölçeksiz
 scale-mode-showall = Tümünü Göster
 scale-mode-force = Zorla
 player-version = Oynatma Sürümü
+player-runtime = Oynatma Süresi
```

Comment:
I'm about 38% sure that at least one of these don't convey the intended original meaning.